iskolivery/app/views/users/_request.html.erb

```diff
@@ -31,7 +31,9 @@ This work is licensed under a Creative Commons Attribution-ShareAlike 4.0 Intern
 			<%= form_tag accept_path do -%>
 			<%= hidden_field_tag :request_id, request.request_id %>
 			<%#= submit_tag 'Accept' %>
-			<button class="ui mini right floated inverted green button" type="submit">Accept</button>
+				<% if request.requester.id != @user.id %>
```

Comment:
add an `else` statement in this block to add more communication to the user that this is their own request